### PR TITLE
Dev lobby script including create game environment

### PR DESCRIPTION
**Pull request by Rivalson11**
[99&15.md](https://github.com/witseie-elen4010/2022-group-lab-011/files/8782060/99.15.md)

### CODE REVIEW

File: sessions/G1qf2gSy2XWHFWbIHpIkZ2IVFzFfihqq.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"cookie":{"originalMaxAge":null,"expires":null,"httpOnly":true,"path":"/"},"ID":2,"__lastAccess":1653597977559}
```
Comment (robinjonker) [inline sessions/G1qf2gSy2XWHFWbIHpIkZ2IVFzFfihqq.json:1]: Can we remove this session file from merging?

File: routes/lobby.js [hunk 2]
```diff
@@ -43,16 +46,51 @@ router.post('/', (req, res) => {
...
+                        .then((pool) => {
+                            return pool.request()
+                             // Select game admins
+                              .input('player_one', playerOne)
+                              .input('player_two', playerTwo)
+                              .input('player_admin', playerAdmin)
+                              .input('word', '')
+                              .query('INSERT INTO dbo.games (player_one, player_two, player_admin, word) VALUES (@player_one, @player_two, @player_admin, @word);')  
+                        })
+                        .then(result => {
```
Comment (robinjonker) [inline routes/lobby.js:88]: Is there a way to refractor this code so we do not have so many nested statements

File: routes/lobby.js [hunk 3]
```diff
@@ -12,6 +12,9 @@ router.post('/', (req, res) => {
     if (!req.session.ID) {
       res.redirect('/login')
       } else {
+    let playerOne
+    let playerTwo
+    let playerAdmin
     const playerRole = req.body.game_role
```
Comment (robinjonker) [inline routes/lobby.js:18]: Do you have a circumstance for if you don't receive the correct information

**Review by robinjonker — CHANGES_REQUESTED** [review]

File: sessions/G1qf2gSy2XWHFWbIHpIkZ2IVFzFfihqq.json (hunk 1 above)
Comment (Rivalson11) [reply sessions/G1qf2gSy2XWHFWbIHpIkZ2IVFzFfihqq.json:1]: we can

File: routes/lobby.js (hunk 2 above)
Comment (Rivalson11) [reply routes/lobby.js:88]: WE can but i recommend we look at creating an issue for code refactoring for sprint 4 due to deadline pressures

File: routes/lobby.js (hunk 3 above)
Comment (Rivalson11) [reply routes/lobby.js:18]: yes defualt values are assigned

File: routes/lobby.js (hunk 2 above)
Comment (robinjonker) [reply routes/lobby.js:88]: Perfect

File: routes/lobby.js (hunk 3 above)
Comment (robinjonker) [reply routes/lobby.js:18]: Okay, we must note this for future bug fixing

**Review by robinjonker — APPROVED** [review]
[code_review_100.md](https://github.com/witseie-elen4010/2022-group-lab-011/files/8782152/code_review_100.md)

**Pull request merged**